zboss/CHANGELOG.rst

```diff
@@ -9,10 +9,10 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
 
-Main branch
-***********
+nRF Connect SDK v1.9.0
+**********************
 
-All the notable changes included in the |NCS| main branch are documented in this section.
+All the notable changes included in the |NCS| v1.9.0 release are documented in this section.
```

Comment:
Please update the sentence in line 30th (possibly 57th) too:
```
For detailed information, see `ZBOSS stack release notes`_ for the main branch.
```
To:
```
For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.9.0 release branch.
```

Reply:
Changed:
- line 30 to For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.9.0.
- line 57 to For detailed information, see `ZBOSS stack release notes`_ for the |NCS| v1.8.0.
Aligning with how it was done for NCS release v1.7.0